src/servers/http_server_v2.cc

```diff
@@ -768,18 +768,22 @@ HTTPAPIServerV2::HandleServerReady(
     err = TRTSERVER_ServerIsReady(server_.get(), &ready);
   }
 
-  RequestStatus request_status;
-  RequestStatusUtil::Create(
-      &request_status, err, RequestStatusUtil::NextUniqueRequestId(),
-      server_id_);
-
-  evhtp_headers_add_header(
-      req->headers_out,
-      evhtp_header_new(
-          kStatusHTTPHeader, request_status.ShortDebugString().c_str(), 1, 1));
-
-  evhtp_send_reply(
-      req, (ready && (err == nullptr)) ? EVHTP_RES_OK : EVHTP_RES_BADREQ);
+  if (err == nullptr) {
```

Comment:
Why is this function called "Ready" when it is both live and ready. Why is the model version called "Health" when it is just ready. Please take some time to think of those coming after you into this code and make it as easy as possible for them to understand what is going on.

Reply:
Sorry. I will make the necessary changes